### ISSUE TITLE
Simplify the output of single value server commands

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#594]
The `server brand` and `server interop` commands show only a single value but add quite some words around that. This could be simplified to showing only the output value.

Examples (from quickstart docs, not verified with latest code):

```
pywbemcli> server brand

Server Brand:
WBEM Server Brand
-------------------
OpenPegasus
```
```
pywbemcli> server interop

Server Interop Namespace:
Namespace Name
----------------
root/PG_InterOp
```

**Comment by KSchopmeyer** [comment]
In dealing with this issue, we are introducing a new concept in output.  Right now we have cimobject and table output.   We want something for these types so that only the data shows up for things like scripting where redirection can be used.

We will make a format format 'text' that is just the text data.

**Comment by andy-maier** [comment]
Addressed by PR #600.
